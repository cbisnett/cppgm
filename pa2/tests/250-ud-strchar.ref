user-defined-literal 'u'_da _da character char 75
invalid 'u'da
user-defined-literal "fsaf"_da _da string array of 5 char 6673616600
simple ; OP_SEMICOLON
invalid "dafa"da
eof
