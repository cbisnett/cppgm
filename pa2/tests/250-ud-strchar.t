'u'_da

'u'da

"fsaf"_da;

"dafa"da

