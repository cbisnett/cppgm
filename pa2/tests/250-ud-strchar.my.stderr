ERROR: invalid user defined suffix: da
ERROR: invalid user defined suffix
